docs/install.rst

```diff
@@ -1,26 +1,126 @@
 
-Using pip
-=========
+Installing miniforge
+====================
+
+We recommend installing Python with `miniforge <https://github.com/conda-forge/miniforge/#miniforge>`__.
+This will install ``conda`` and automatically configure the default channel (a channel is a remote software repository) to be ``conda-forge``.
+
+First, download the installer for your system and architecture from the links below:
+
+.. grid:: 3
+
+    .. grid-item-card:: Linux
+
+        `x86-64 <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh>`__
+
+        `aarch64 <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-aarch64.sh>`__
+
+        `ppc64le <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-ppc64le.sh>`__
+
+    .. grid-item-card:: Windows
+        :link: https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe
+
+        `x86-64 <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Windows-x86_64.exe>`__
+
+    .. grid-item-card:: Mac
+
+        `x86-64 <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-x86_64.sh>`__
+
+        `arm64 (Apple Silicon) <https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-MacOSX-arm64.sh>`__
+
+Then select your platform to install miniforge:
+
+.. tab-set::
+
+    .. tab-item:: Linux & Mac
+        :sync: platform
+
+        Linux & Mac Run the script downloaded above, with
+        ``bash <filename>``. The following should work:
+
+        .. code-block:: console
+
+            bash Miniforge3-$(uname)-$(uname -m).sh
+
+        Once the installer has completed, close and reopen your terminal.
+
+    .. tab-item:: Windows
+        :sync: platform
+
+        Double click the executable file downloaded from
+        the links above.
+
+        Once the installer has completed you should have a new "miniforge
+        Prompt" entry in your start menu.
+
+In a new terminal (miniforge Prompt on Windows) run ``conda list`` to test that the install has worked.
+
+Using an Existing Conda Installation
+====================================
+
+If you already have a conda installation you can use it to install astropy.
+We do not recommend reusing an anaconda installation, as the default packages provided by Anaconda Inc. are often out dated and can require a license for use at a large organisaiton.
+If you have installed miniforge or anaconda you can add ``-c conda-forge`` to all installation commands to install packages from `conda-forge <https://conda-forge.org>`__.
+
+
+Installing astropy
+==================
+
+To install ``astropy``, start by launching a terminal (under a UNIX-like system) or the miniforge Prompt (under Windows).
+Now we will create and activate a new virtual environment to install ``astropy`` into:
+
+.. code-block:: bash
+
+    $ conda create --name astropy
+    $ conda activate astropy
```

Comment:
To avoid confusing first-time users, I think it might be preferable to use a distinctive name for then environment itself
```suggestion
    $ conda create --name astropy-env
    $ conda activate astropy-env
```